src/modules/core-milestone/core-milestone.tsx

```diff
@@ -0,0 +1,289 @@
+                props.hasAttendedClass201 && props.hasSignedMaturityCovenant
+                  ? props.isWhite === true
+                    ? '#fff'
+                    : props.colorMilestone
+                  : props.hasAttendedClass201
+                  ? 'transparent'
+                  : '#dde1e4'};
+              borderRadius: 1px;
+              borderWidth: 0.5px;
+              borderColor:${props =>
+                props.hasAttendedClass201 && props.hasSignedMaturityCovenant
+                  ? 'transparent'
+                  : props.hasAttendedClass201
+                  ? props.isWhite === true
+                    ? '#fff'
+                    : props.colorMilestone
+                  : 'transparent'};
+              transform:rotate(135deg);
+`;
+
+// Bottom hasAttendedClass401
+const StyledClassHasAttendedClass401Wrapper = Styled.View<IProps>`
+height: ${Number(minorScale(1)) + 1}px;
+              width: ${Number(minorScale(1)) + 1}px;
+              marginTop: 4;
+              backgroundColor:${props =>
+                props.hasAttendedClass401 && props.hasSignedMissionCovenant
+                  ? props.isWhite === true
+                    ? '#fff'
+                    : props.colorMilestone
+                  : props.hasAttendedClass401
+                  ? 'transparent'
+                  : '#dde1e4'};
+              borderRadius: 1px;
+              borderWidth: 0.5px;
+              borderColor:${props =>
+                props.hasAttendedClass401 && props.hasSignedMissionCovenant
+                  ? 'transparent'
+                  : props.hasAttendedClass401
+                  ? props.isWhite === true
+                    ? '#fff'
+                    : props.colorMilestone
+                  : 'transparent'};
+              transform:rotate(135deg);
+`;
+
+// Rigth hasAttendedClass101
+const StyledClassHasAttendedClass101Wrapper = Styled.View<IProps>`
+height: ${Number(minorScale(1)) + 1}px;
+            width: ${Number(minorScale(1)) + 1}px;
+            marginLeft: -1px;
+            backgroundColor:${props =>
+              props.hasAttendedClass101 && props.hasSignedMembershipAgreement
+                ? props.isWhite === true
+                  ? '#fff'
+                  : props.colorMilestone
+                : props.hasAttendedClass101
+                ? 'white'
+                : '#dde1e4'};
+            borderRadius: 1px;
+            borderWidth: 0.5px;
+            borderColor:${props =>
+              props.hasAttendedClass101 && props.hasSignedMembershipAgreement
+                ? 'transparent'
+                : props.hasAttendedClass101
+                ? props.isWhite === true
+                  ? '#fff'
+                  : props.colorMilestone
+                : 'transparent'};
+            transform: rotate(135deg);
+`;
+
+const StyledAcceptChristWrapper = Styled.View`
+width: ${minorScale(3)};
+height: ${minorScale(3)};
+margin-horizontal: ${minorScale(1)};
```

Comment:
Can we please avoid these repetitive statements?

Reply:
I have fixed it.